bat/tests/06-test-format-bump/run.bats

```diff
@@ -1,14 +1,48 @@
 #!/usr/bin/env bats
 
 # shared test functions
-#load ../../lib/mixerlib
+load ../../lib/mixerlib
 
-#setup() {
-#  global_setup
-#}
+setup() {
+  global_setup
+}
 
-# TODO: fill out once auto format bump implemented
-#@test "Create initial mix 10" {
-#}
+# TODO: This test should run 'mixer build upstream-format' instead
+# once the tooling required to run custom test docker images for
+# is ready.
+@test "Perform upstream format bump on last format boundary" {
+  upstreamver=$(get-last-format-boundary)
+  mixer-init-stripped-down $upstreamver 10
+
+  #Create initial version in the old format
+  mixer-build-all > $LOGDIR/build_all.log
+
+  #Update to new upstream in the new format
+  mixer-upstream-update $(($upstreamver + 10)) > $LOGDIR/upstream_update.log
+
+  #Build +20
+  mixer-build-format-bump-new > $LOGDIR/build_formatbump_new.log
+
+  #Reset upstream for +10 build
+  echo $upstreamver > upstreamversion
+
+  #Build +10
+  mixer-build-format-bump-old > $LOGDIR/build_formatbump_old.log
+
+  #restore upstrean
+  mv upstreamversion.bump upstreamversion
+
+  #check if LAST_VER is set to +20
+  test $(cat update/image/LAST_VER) -eq 30
```

Comment:
In bash, you can make this slightly more efficient by saying

```test $(< update/image/LAST_VER) -eq 30```

not enough to make it worth changing if this was the only thing to do.

Reply:
Updated, thanks.